lib/puma/binder.rb

```diff
@@ -50,14 +50,6 @@ def env(sock)
 
     def close
       @ios.each { |i| i.close }
-      @unix_paths.each do |i|
```

Comment:
This doesn't fix the problem, as bind paths are still not removed after program stop.

Reply:
A few days ago, I was really clear on what was where in terms of Binder `@ios`, `@unix_paths`, and `@listeners`.  I should have added some comments...

Paths are closed in `Binder#close_listeners`, They aren't in `Binder#close`

Reply:
[See this assertion I've added to master.](https://github.com/puma/puma/commit/2ae66ce61c6ab0ef4e4df15b324eaf953667979b#diff-e405df74eaf050d8fc90276c05c35b39R17) Anything that purports to fix this problem needs to pass this test (which is failing on master).